crates/shared/src/order_quoting.rs

```diff
@@ -488,14 +510,15 @@ impl OrderQuoter {
             } => (trade_estimate.out_amount, *buy_amount),
         };
         let fee_parameters = FeeParameters {
-            gas_amount: trade_estimate.gas as _,
+            gas_amount: trade_estimate.gas as f64 + signature_gas_fee.to_f64_lossy(),
```

Comment:
I don't think we should include this here. This means that the `FeeParameters` are tied to a specific signing scheme and will be store in the database that way.

In fact, I think if we implement `FeeParameters::subsidized_with_additional_cost` function, then the additional `verification_gas_limit` will be added when computing the final `Quote::fee_amount` and not `FeeParameters::gas_amount`.

Reply:
Removed.